sentry-ruby/lib/sentry/rack/capture_exceptions.rb

```diff
@@ -62,7 +63,17 @@ def capture_exception(exception, env)
       end
 
       def start_transaction(env, scope)
-        options = { name: scope.transaction_name, source: scope.transaction_source, op: transaction_op }
+        options = {
+          name: scope.transaction_name,
+          source: scope.transaction_source, 
+          op: transaction_op
+        }
+        
+        # Tell Sentry to not sample this transaction if this is an HTTP OPTIONS or HEAD request.
+        if IGNORED_HTTP_METHODS.include?(env["REQUEST_METHOD"])
+          options.merge!(sampled: false)
```

Comment:
I wonder if we should early return here instead so later computations (which are plenty of them) are avoided.

@sl0thentr0py WDYT?

Reply:
yes I was also about to write the same but got distracted, we should just return early

Reply:
I'm on it!

Reply:
Alright, catching up. 

If I read the code correctly, then if we bail out and return nil early, we won't start a transaction. The only side effect that I see is that we won't set the transaction on that scope, but we'll still track errors as usual, and we'll set the transaction name based on the request path.

I _think_ this is fine. Because if we do go and make the transaction with `@sampled = false`, it won't be much use anyway questionmark.

Pushing up the change now.